<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sajadminsaveDialog</class>
 <widget class="QDialog" name="sajadminsaveDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>630</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:rgb(0, 0, 0)</string>
  </property>
  <widget class="QLabel" name="label_indentitysaj">
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>180</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>  identity</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_identitysaj">
   <property name="geometry">
    <rect>
     <x>224</x>
     <y>180</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background:rgb(255, 255, 255)</string>
   </property>
   <property name="placeholderText">
    <string>         book or booked</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_savesaj">
   <property name="geometry">
    <rect>
     <x>388</x>
     <y>300</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;
background:rgb(170, 0, 0)</string>
   </property>
   <property name="text">
    <string>save</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_eidsaj">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>250</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>   eid</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_namesaj">
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>50</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>  Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_namesaj">
   <property name="geometry">
    <rect>
     <x>224</x>
     <y>50</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background:rgb(255, 255, 255)</string>
   </property>
   <property name="placeholderText">
    <string>          chittagong     resort .</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_eidsaj">
   <property name="geometry">
    <rect>
     <x>224</x>
     <y>250</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background:rgb(255, 255, 255)</string>
   </property>
   <property name="placeholderText">
    <string>          ctg01</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_descriptionsaj">
   <property name="geometry">
    <rect>
     <x>224</x>
     <y>110</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background:rgb(255, 255, 255)</string>
   </property>
   <property name="placeholderText">
    <string>           Agrabad, Chittagong</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_descriptionsaj">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 10pt &quot;MS Shell Dlg 2&quot;;
background:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>   description</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_updatesaj">
   <property name="geometry">
    <rect>
     <x>224</x>
     <y>300</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;
background:rgb(170, 0, 0)</string>
   </property>
   <property name="text">
    <string>update</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
